apps/ledger-live-mobile/src/screens/PostOnboarding/PostOnboardingHub.tsx

```diff
@@ -56,9 +48,7 @@ const PostOnboardingHub: React.FC<StackScreenProps<Props>> = ({
   const dispatch = useDispatch();
   const { t } = useTranslation();
   const { lastActionCompleted, actionsState } = usePostOnboardingHubState();
-  const [popupOpened, setPopupOpened] = useState(true);
-  const { actionCompletedHubTitle, actionCompletedPopupLabel } =
-    lastActionCompleted ?? {};
+  const { actionCompletedHubTitle } = lastActionCompleted ?? {};
```

Comment:
Do we really need `??` here? Instead of `||` that more people knows

Reply:
👍